packages/block-library/src/comment-author-name/block.json

```diff
@@ -20,7 +20,7 @@
 			"type": "string"
 		}
 	},
-	"usesContext": [ "commentId" ],
+	"usesContext": [ "__client", "commentId" ],
```

Comment:
In the final API, block authors shouldn't have to specify the `__client` context here. Instead, we should derive it from a `block.json` flag like `hydrate` or `interactive`.